groups/sig-release/groups.yaml

```diff
@@ -241,6 +242,7 @@ groups:
       ReconcileMembers: "true"
     owners:
       - adolfo.garcia@uservers.net
+      - cicih@google.com
```

Comment:
@cici37  lets drop this one for now. This would give you access to some comms that I don't think are applicable at this stage.